app/views/questionnaire.py

```diff
@@ -42,11 +43,13 @@
 @questionnaire_blueprint.before_request
 @login_required
 def check_survey_state():
```

Comment:
I think this name is due a change as its not just `check_survey_state` its more like a generic pre request setup and checks.